### DISCUSSION TITLE
Preload data for next word prediction

### DISCUSSION THREAD

**Discussion by bnika** [AnySoftKeyboard/AnySoftKeyboard#3484] [Q&A]
I would like to preload bigrams with frequencies to enable solid next word prediction from the beginning, i.e. not only based on usage data. Is there any possibility to do that in a way foreseen by the ASK developers?

**Comment by menny** [comment]
It's possible to do that, for sure. But, the issue is that once we ship this large database, we can not change it.
We'll need to copy that somewhere writable and work from there. So, that initial database is just dead weight

**Comment by menny** [comment]
It's possible to do that, for sure. But, the issue is that once we ship this large database, we can not change it.
We'll need to copy that somewhere writable and work from there. So, that initial database is just dead weight

**Comment by nicoursi** [comment]
What about shipping it as a pack and prompting the user to uninstall it after coping the data?
